mmv1/third_party/terraform/resources/resource_tags_location_tag_bindings.go.erb

```diff
@@ -0,0 +1,349 @@
+			},
+			"location": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Optional: true,
+				Description: `The geographic location where the transfer config should reside.
+Examples: US, EU, asia-northeast1. The default value is US.`,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The generated id for the TagBinding. This is a string of the form: 'tagBindings/{full-resource-name}/{tag-value-name}'`,
+			},
+		},
+		UseJSONNumber: true,
+	}
+}
+
+func resourceTagsLocationTagBindingCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	userAgent, err := generateUserAgentString(d, config.userAgent)
+	if err != nil {
+		return err
+	}
+
+	obj := make(map[string]interface{})
+	parentProp, err := expandNestedTagsLocationTagBindingParent(d.Get("parent"), d, config)
+	if err != nil {
+		return err
+	} else if v, ok := d.GetOkExists("parent"); !isEmptyValue(reflect.ValueOf(parentProp)) && (ok || !reflect.DeepEqual(v, parentProp)) {
+		obj["parent"] = parentProp
+	}
+	tagValueProp, err := expandNestedTagsLocationTagBindingTagValue(d.Get("tag_value"), d, config)
+	if err != nil {
+		return err
+	} else if v, ok := d.GetOkExists("tag_value"); !isEmptyValue(reflect.ValueOf(tagValueProp)) && (ok || !reflect.DeepEqual(v, tagValueProp)) {
+		obj["tagValue"] = tagValueProp
+	}
+
+	url, err := replaceVars(d, config, "{{TagsLocationBasePath}}tagBindings")
+	log.Printf("url for TagsLocation: %s", url)
+	if err != nil {
+		return err
+	}
+
+	log.Printf("[DEBUG] Creating new LocationTagBinding: %#v", obj)
+	billingProject := ""
+
+	// err == nil indicates that the billing_project value was found
+	if bp, err := getBillingProject(d, config); err == nil {
+		billingProject = bp
+	}
+
+	res, err := sendRequestWithTimeout(config, "POST", billingProject, url, userAgent, obj, d.Timeout(schema.TimeoutCreate))
+	if err != nil {
+		return fmt.Errorf("Error creating LocationTagBinding: %s", err)
+	}
+
+	// Store the ID now
+	id, err := replaceVars(d, config, "{{location}}/{{name}}")
+	if err != nil {
+		return fmt.Errorf("Error constructing id: %s", err)
+	}
+	d.SetId(id)
+
+	// Use the resource in the operation response to populate
+	// identity fields and d.Id() before read
+	var opRes map[string]interface{}
+	err = tagsOperationWaitTimeWithResponse(
+		config, res, &opRes, "Creating LocationTagBinding", userAgent,
+		d.Timeout(schema.TimeoutCreate))
+	if err != nil {
+		// The resource didn't actually create
+		d.SetId("")
+		return fmt.Errorf("Error waiting to create LocationTagBinding: %s", err)
+	}
+
+	if _, ok := opRes["tagBindings"]; ok {
+		opRes, err = flattenNestedTagsLocationTagBinding(d, meta, opRes)
+		if err != nil {
+			return fmt.Errorf("Error getting nested object from operation response: %s", err)
+		}
+		if opRes == nil {
+			// Object isn't there any more - remove it from the state.
```

Comment:
Comment says remove it from state, but code doesn't remove it from state

Reply:
fixed